pkg/apis/management.cattle.io/v3/authz_types.go

```diff
@@ -375,7 +375,8 @@ type ClusterRoleTemplateBinding struct {
 	// +optional
 	GroupPrincipalName string `json:"groupPrincipalName,omitempty" norman:"noupdate,type=reference[principal]"`
 
-	// ClusterName is the name of the cluster to which a subject is added. Immutable.
+	// ClusterName is the metadata.name of the cluster (not displayName) to which a subject is added.
```

Comment:
Nit: I think the `(not displayName)` part is a bit superflous and might confuse people, I'd recommend just sticking to 
```suggestion
	// ClusterName is the metadata.name of the cluster to which a subject is added.
```